.github/workflows/build.yml

```diff
@@ -24,24 +24,30 @@ jobs:
           - "6.0"
           - "6.1"
           - main
+        include:
+          - ruby: "3.1"
+            rails: "7.0"
+          - ruby: "3.1"
+            rails: "6.1"
         exclude:
           - ruby: 2.5
             rails: main
           - ruby: 2.6
             rails: main
           - ruby: jruby-9.2.16.0
             rails: main
-          - ruby: 3.0
+          - ruby: "3.0"
             rails: "5.0"
-          - ruby: 3.0
+          - ruby: "3.0"
             rails: "5.1"
-          - ruby: 3.0
+          - ruby: "3.0"
             rails: "5.2"
 
     runs-on: 'ubuntu-latest'
 
     env:
       BUNDLE_GEMFILE: gemfiles/${{ matrix.rails }}.gemfile
+      RUBYOPT: "--disable-error_highlight"
```

Comment:
I'm curious what error you were seeing that led you to add this. Is everyone running workflows with Ruby 3.1 going to need this?

Reply:
You can find the discussion here - https://github.com/rspec/rspec-expectations/issues/1338 .

And here's what it looked like for `factory_bot`:

<img width="1270" alt="Screen Shot 2022-01-14 at 10 44 47 AM" src="https://user-images.githubusercontent.com/421488/149568558-3b689abe-6d5b-472b-9232-d7db7ac1524d.png">

Reply:
As you can see in `rspec-expectations` I submitted a PR to resolve this issue in matchers that was just merged today.

Reply:
👍🏻 Thank you for explaining, and for fixing the underlying problem!